lhotse/supervision.py

```diff
@@ -251,12 +254,49 @@ def from_segments(segments: Iterable[SupervisionSegment]) -> 'SupervisionSet':
     def from_dicts(data: Iterable[Dict]) -> 'SupervisionSet':
         return SupervisionSet.from_segments(SupervisionSegment.from_dict(s) for s in data)
     
-    def add_alignments_from_ctm(ctm) -> 'SupervisionSet':
+    def add_alignments_from_ctm(self, ctm_file: Pathlike, type: str = 'word') -> 'SupervisionSet':
         """
+        Add alignments from CTM file to the supervision set.
         
-        """
-        pass
-
+        :param ctm: Path to CTM file.
+        :param type: Alignment type (optional, default = `word`).
+        :return: A new SupervisionSet with AlignmentItem objects added to the segments.
+        """
+        ctm_words = []
+        with open(ctm_file, 'r') as f:
+            for line in f:
+                reco_id, channel, start, duration, symbol = line.strip().split()
+                ctm_words.append((reco_id, channel, float(start), float(duration), symbol))
```

Comment:
shouldn't `channel` be an int?

Reply:
(at least in Lhotse we always map them to ints and expect int, so be careful there)

Reply:
Ideally, yes. But sometimes they are also denoted by `A`, `B`, etc. in CTM files.

Reply:
yes, channels do not have to be int